commit/merkleroot/rmn/config.go

```diff
@@ -0,0 +1,37 @@
+package rmn
+
+import (
+	"crypto/ed25519"
+
+	mapset "github.com/deckarep/golang-set/v2"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+)
+
+// Config contains the RMN configuration required by the plugin and the RMN client in a single struct.
+type Config struct {
+	Home   RMNHomeConfig
+	Remote RMNRemoteConfig
```

Comment:
A split with a struct for Home config and another one for Remote config might help the way we are fetching the data. But `RMNNodeInfo` is actually having data from both (like SignReportsAddress). So we might have to have 2 type of RmnNodeInfo struct: one for Home and one for Remote.

Reply:
ah interesting, i believe we can merge, like this. and apply relevant config changes in a follow-up